setup.cfg

```diff
@@ -13,5 +13,6 @@ logging-level=INFO
 [flake8]
 max-line-length = 150
 max-complexity = 30
+ignore = E501
```

Comment:
@jmchilton Please feel free to remove 

'''
                                   "Christophe Antoniewski <drosofff@gmail.com>\n"
                                   "https://github.com/ARTbio/ansible-artimed/tree/master/extra-files/generate_tool_list_from_ga_workflow_files.py")
```
from generate_tool_list_from_ga_workflow_files.py. It is non sense as well as deprecated.